designer/report-designer-extension-pentaho/pom.xml

```diff
@@ -74,5 +74,14 @@
       <artifactId>hamcrest-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <scope>compile</scope>
```

Comment:
Doesn't need to specify `<scope>compile</scope>`

Reply:
Removed.